src/constants.js

```diff
@@ -15,15 +15,18 @@ export const FPTI_STATE = {
   PXP: ("PXP_CHECK": "PXP_CHECK"),
 };
 
-export const FRAUDNET_URL = {
+type FraudnetUrl = {|
+  [$Values<typeof ENV>]: string,
+|};
+
+export const FRAUDNET_URL: FraudnetUrl = {
   [ENV.LOCAL]:
-    "https://cdn-latest.static.engineering.dev.paypalinc.com/rdaAssets/fraudnet/async/fb-raw.js",
+    "https://cdn-latest.static.engineering.dev.paypalinc.com/qaRdaAssets/fraudnet/async/fb-raw.js",
```

Comment:
TODO: Confirm if this is right, I think the version in `main` for these would actually be correct.